src/app/components/introduction.tsx

```diff
@@ -13,15 +16,30 @@ export default function Introduction() {
   `;
 
   return (
-    <div className="fixed top-0 left-0 w-screen h-screen flex items-center justify-center -z-10">
-      <div className="p-5">
-        <h2 className="text-center font-serif text-2xl">Joe Owen</h2>
-        <div className="max-w-prose p-5">
-          <pre className="pb-5 font-sans whitespace-normal">{introTextSummary}</pre>
-          <pre className="font-sans whitespace-normal">{introTextCollaborate}</pre>
+    <div className="fixed top-0 left-0 w-full h-full flex items-center justify-center -z-10 bg-vertical lg:bg-horizontal bg-cover bg-center">
+      <div className="flex flex-col items-center p-3">
+        <Image
+          src={profilePicture}
+          alt="Profile picture"
+          className="rounded-full w-28"
+        />
+        <h2 className="p-5 font-serif text-2xl">Joe Owen</h2>
+        <div className="max-w-prose">
+          <IntroParagraph>{introTextSummary}</IntroParagraph>
+          <IntroParagraph>{introTextCollaborate}</IntroParagraph>
         </div>
       </div>
       <pre className="fixed bottom-5 font-sans">{"Scroll to view more"}</pre>
     </div>
   )
+}
+
+type IntroParagraphProps = {
+  children: string;
+}
+
+function IntroParagraph({ children }:IntroParagraphProps ) {
+  return (
+    <pre className="p-2 mb-3 font-sans whitespace-normal bg-black/75">{children}</pre>
```

Comment:
Nice, I like it!

Reply:
Thanks! Felt pretty lame making a component just for a bit of text. But I think it'll be a lot easier moving forward to just change the styling of the one component

Reply:
Yeah, it keeps it nice and clean though. You could always name it something more generic or even keep the naming as it is and use it on other pages to keep the styling consistent